docs/academy/exercise1.md

```diff
@@ -47,7 +49,10 @@ Start by creating a new workspace, installing the dependencies, and cloning all
         ```
 
     === "Windows"
-        Download and run the [installer](https://developer.arm.com/-/media/Files/downloads/gnu-rm/10-2020q4/gcc-arm-none-eabi-10-2020-q4-major-win32.exec) to install `arm-none-eabi-gcc` and `arm-none-eabi-gdb`.
+        !!! note
```

Comment:
Why is this a note? It's in the Windows tab, it doesn't and shouldn't be a note.